config/metrics.js

```diff
@@ -113,6 +113,10 @@ module.exports = {
 		extract: (item) => !!getBPRules(item)?.allowsDeletions,
 		permissions: ['ADMIN'],
 	},
+	AllowsCocs: {
+		extract: (item) => !!getBPRules(item)?.AllowsCocs,
+		permissions: ['ADMIN'],
```

Comment:
the CoC is just a file, so anyone with WRITE access or above should be able to enable it

Reply:
Updated.